policy/modules/admin/logrotate.te

```diff
@@ -121,6 +129,10 @@ logging_send_syslog_msg(logrotate_t)
 logging_send_audit_msgs(logrotate_t)
 logging_exec_all_logs(logrotate_t)
 
+tunable_policy(`logrotate_manage_audit_log',`
+	logging_manage_audit_log(logrotate_t)
+')
```

Comment:
Sorry, I missed this in my first read, this block should be after the distro_debian block below (before the optionals)